Check Point - Playbook and Custom Connector

chkp-jguo opened the pull request:
Fixes #

## Proposed Changes

  - Add Check Point logo to logo directory
  - Add Check Point / Playbook folder to Solutions directory
  - Add Check Point / Playbook folder which contains Check Point custom connector and playbook

sarah-yo commented on Solutions/Check Point/Playbooks/README.md:
```diff
@@ -0,0 +1,231 @@
+
+For more information see
+[Check Point Management API](https://sc1.checkpoint.com/documents/latest/APIs/#introduction~v1.7%20) and 
+[Logic App Overview](https://azure.microsoft.com/en-us/services/logic-apps/) 
```
Remove localisation "en-us" from URL

chkp-jguo replied:
Removed localization - per https://github.com/Azure/Azure-Sentinel/pull/2052/commits/2b05dc45e52c19bc1a42eeaea62226fafa221843

sarah-yo reviewed (changes requested):
Hello, please can you do the following:

-  remove any localisation from your URLs (I have marked one example but please check your whole readme)
- Add a "Deploy to Azure Gov button and ARM template
- Update your "Deploy to Azure button image with the most recent version as can be seen on other readmes in this repo

Thanks!

chkp-jguo commented:
> Hello, please can you do the following:
> 
> * remove any localisation from your URLs (I have marked one example but please check your whole readme)
> * Add a "Deploy to Azure Gov button and ARM template
> * Update your "Deploy to Azure button image with the most recent version as can be seen on other readmes in this repo
> 
> Thanks!

1. Removed localization in readme
2. Added deploy to azure gov and arm template
3. Updated deploy to azure button image

chkp-jguo commented:
will submit a new one.
